circle.yml

```diff
@@ -275,12 +216,35 @@ jobs:
       - *build
       - *save-deps-cache
 
+  linux-shared-evm2wasm:
```

Comment:
Don't you want to merge this tests with `linux-*-shared-release` or `linux-*-shared-coverage`?

Reply:
I don't want to include it in coverage.

Reply:
So add it to  linux-clang-shared-release